tools/bdist_all_wheels.py

```diff
@@ -34,14 +31,25 @@
 import setuptools
 
 setuptools.setup(
-   name="odoo-addons-oca-{repo}",
+   name="odoo{series}-addons-oca-{repo}",
    description="Meta package for OCA {repo} Odoo addons",
    version="{branch}.{date}",
    install_requires={install_requires},
 )
 """
 
 
+def branch_to_series(branch):
+    if os.environ.get('SETUPTOOLS_ODOO_LEGACY_MODE'):
+        return ''
+    if branch.startswith('8.0'):
+        return '8'
+    elif branch.startswith('9.0'):
+        return '9'
```

Comment:
We should include here 10 and 11 as well as we have clear that those 2 will be in the future (or not? :stuck_out_tongue_winking_eye: )